src/main/resources/application.conf

```diff
@@ -86,8 +86,8 @@ db-evolution-config {
 }
 
 akka-http-cors {
-  allowed-origins = ["http://localhost:3000"]
-  allowed-origins = ${?FE_URL}
+  allowed-origins = "*"
```

Comment:
it's will work for anything or just domain scalac.io ? If first maybe better option will be creating for domain + localhost

Reply:
this will be used only wehn running locally by devs (FE + BE) I think blocking origins on this level was not a good idea